internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -462,8 +487,11 @@ func (vm *VM) transfer(debitFrom address.Address, creditTo address.Address, amou
 	ctx := context.Background()
 
 	// retrieve debit account
-	fromActor, err := vm.state.GetActor(ctx, debitFrom)
+	fromActor, found, err := vm.state.GetActor(ctx, debitFrom)
 	if err != nil {
+		panic(err)
```

Comment:
FWIW I think it would be more helpful to propagate an error from this method, deferring any panic or handling to higher up.

Reply:
Yes, there are a few places now that should change to error instead of panic. 
I'll see to this on another PR.